reference/cli/admin_utilities/hawqscp.html.md.erb

```diff
@@ -59,7 +59,7 @@ Overrides copying configuration files to a host on which SSH validation fails. I
 </dd>
 
 <dt>-J \<character\>  </dt>
-<dd>The `-J` option allows you to specify a single character to substitute for the \<hostname\> in the `copy from` and `copy               to` destination strings. If `-J` is not specified, the default substitution character is an equal sign (`=`).</dd>
+<dd>The `-J` option allows you to specify a single character to substitute for the \<hostname\> in the \<file\_to\_copy\> and \<copy\_to\_path\> destination strings. If `-J` is not specified, the default substitution character is an equal sign (`=`).</dd>
```

Comment:
In previous edit, file_to_copy and copy_to_path use code format.  This should be consistent across the board.